datafusion/optimizer/src/propagate_empty_relation.rs

```diff
@@ -120,14 +120,13 @@ impl OptimizerRule for PropagateEmptyRelation {
                         Ok(LogicalPlan::Projection(Projection::new_from_schema(
                             Arc::new(child),
                             optimized_children_plan.schema().clone(),
-                            union.alias.clone(),
+                            None
```

Comment:
@jackwener  After  https://github.com/apache/arrow-datafusion/commit/4dcf9857c0aad3d11050037bd46889a54f83a394 
no field `alias` on `Union`, 

Is there ok pass `none` here?

Reply:
It's due to `Bors` problem.😂